graphene_linked_events/schema.py

```diff
@@ -291,9 +293,16 @@ class Query:
     def resolve_event(parent, info, **kwargs):
         if kwargs.get("include"):
             params = {"include": ",".join(kwargs["include"])}
-            response = api_client.retrieve("event", kwargs["id"], params=params)
+            response = api_client.retrieve(
+                "event",
+                kwargs["id"],
+                params=params,
+                auth_header=info.context.user.is_staff,
```

Comment:
Dont know about the python conventions but this ```auth_header``` could be  something like ```is_staff```. As it now tells to provide the auth header. Or we also could pass the user to client so that checks could happen in single point.

Reply:
Good point! Updated